internal/ui/termstatus/status_test.go

```diff
@@ -19,17 +19,40 @@ func TestTruncate(t *testing.T) {
 		{"foo", 0, ""},
 		{"foo", -1, ""},
 		{"Löwen", 4, "Löwe"},
-		{"あああああああああ/data", 10, "あああああ"},
-		{"あああああああああ/data", 11, "あああああ"},
```

Comment:
I think we should also keep this test, which checks that a unicode character is not split in half.

Reply:
Good point. Restored a version of it.